specification/vmwarecloudsimple/resource-manager/readme.go.md

```diff
@@ -0,0 +1,19 @@
+## Go
+
+These settings apply only when `--go` is specified on the command line.
+
+```yaml $(go)
+go:
+  license-header: MICROSOFT_APACHE_NO_VERSION
+  namespace: vmwarecloudsimple
+  clear-output-folder: true
+```
+
```

Comment:
Missing section `Go multi-api`

Reply:
@jhendrixMSFT thanks for review, comments are addressed